src/backend/joanie/payment/models.py

```diff
@@ -390,6 +394,9 @@ class CreditCard(BaseModel):
         on_delete=models.CASCADE,
     )
     is_main = models.BooleanField(_("main"), default=False)
+    payment_provider = models.CharField(
+        _("payment provider"), max_length=50, null=True, blank=True
```

Comment:
To argue with @jbpenrath suggestion to not set this field as nullable, you should check in the `clean` method if this field is not `None` while we don't make this field required. WDYT ?

Reply:
And you should add a migration, adding the current payment backend provider name in this field already existing records